tests/integration_test/services_test.py

```
@@ -6904,6 +6904,312 @@
+    CollectionData(
+        collection_name=catalog_images_collection,
+        data=fallback_data,
+        user="integration@demo.ai",
+        bot=pytest.bot,
+        status=True,
+        timestamp=datetime.utcnow()
+    ).save()
+
+    CatalogSyncLogs(
+        execution_id=str(ObjectId()),
+        raw_payload={"test":"test"},
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)